doc/usage/tutorial.md

```diff
@@ -2,53 +2,99 @@
 
 This doc will show how to run `kube-batch` as a kubernetes batch scheduler. It is for [master](https://github.com/kubernetes-sigs/kube-batch/tree/master) branch.
 
-## 1. Pre-condition
+## Prerequisite
 To run `kube-batch`, a Kubernetes cluster must start up. Here is a document on [Using kubeadm to Create a Cluster](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/). Additionally, for common purposes and testing and deploying on local machine, one can use Minikube. This is a document on [Running Kubernetes Locally via Minikube](https://kubernetes.io/docs/getting-started-guides/minikube/). Besides, you can also use [kind](https://github.com/kubernetes-sigs/kind) which is a tool for running local Kubernetes clusters using Docker container "nodes".
 
 `kube-batch` need to run as a kubernetes scheduler. The next step will show how to run `kube-batch` as kubernetes scheduler quickly. Refer [Configure Multiple Schedulers](https://kubernetes.io/docs/tasks/administer-cluster/configure-multiple-schedulers/) to get more details.
 
-## 2. Config kube-batch for Kubernetes
+## Install kube-batch for Kubernetes
 
-### (1) kube-batch image
-
-An official kube-batch image is provided and you can download it from [DockerHub](https://hub.docker.com/r/kubesigs/kube-batch/). The version is `v0.4.2` now.
-
-```bash
-# docker pull kubesigs/kube-batch:v0.4.2
-```
-
-### (2) Create a Kubernetes Deployment for kube-batch
-
-#### Download kube-batch
+### Download kube-batch
 
 ```bash
 # mkdir -p $GOPATH/src/github.com/kubernetes-sigs/
 # cd $GOPATH/src/github.com/kubernetes-sigs/
 # git clone http://github.com/kubernetes-sigs/kube-batch
 ```
 
-#### Deploy `kube-batch` by Helm
+### Deploy kube-batch by Helm
 
-Run the `kube-batch` as kubernetes scheduler
+Run `kube-batch` as kubernetes scheduler. The latest kube-batch version is `v0.4.2`.
 
 ```bash
 # helm install $GOPATH/src/github.com/kubernetes-sigs/kube-batch/deployment/kube-batch --namespace kube-system
 ```
 
-Note: If there is an error `Error: apiVersion "scheduling.incubator.k8s.io/v1alpha1" in kube-batch/templates/default.yaml is not available`, please update your helm to latest version and try it again.
```

Comment:
Please rebase your code to latest version. This note are not supposed to delete :)

Reply:
Hi @hex108 , this has not been deleted. It's moved to section FAQs. I notice tutorial have lots of notes and not all users will encounter same issues. For these kind of "notes", I move to FAQ and people could look up solution based on their needs.

Reply:
Sorry, it seems be moved to FAQ section.